config/postgres/test_setup/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+
+services:
+  postgresql:
+    image: transporter_postgresql:latest
```

Comment:
I had to change this line to `image: transporter_postgres:latest` to get it to run.

Reply:
Yeah, was wondering why it would work on CI ... turns out it wasn't enabled, whoops. Pushed changes to fix it.